src/app/AddCard.tsx

```diff
@@ -1,18 +1,25 @@
 import { useContext, useState } from 'react'
 import { CardInfoContext, UpdateCardInfoContext } from '../context/paymentContext'
-import { CardBox, CardCvc, CardName, CardNumber, CardPassword, CardType, CardDate } from '../components'
+import { CardBox, CardCvc, CardName, CardNumber, CardPassword, CardDate, CardType } from '@/components'
 import { StepProps } from './Payments'
 import { CardListContext } from '../context/cardListContext'
 import ui from '../styles/index.module.css'
 
 const AddCard = ({ onStep }: StepProps) => {
   const cardInfo = useContext(CardInfoContext)
   const updateCardInfo = useContext(UpdateCardInfoContext)
-
   const cardList = useContext(CardListContext)
 
   const [isShowModal, setIsShowModal] = useState(false)
 
+  const handleNextButtonClick = () => {
+    if (!cardInfo.cardType?.name) return setIsShowModal((state) => !state)
+
+    if (Object.keys(cardInfo).length < 7) return alert('모든 정보를 입력해주세요.')
```

Comment:
```suggestion
    if (!cardInfo.cardType?.name) {
        setIsShowModal((state) => !state)
        return
    }

    if (Object.keys(cardInfo).length < 7) {
        alert('모든 정보를 입력해주세요.')
        return
    }
```
함수 실행 중단을 위해 return을 하는 경우 함수 실행 결과를 반환값으로 두는건 추천드리지않아요!

`return someFunction()`의 의미는 일반적으로 `someFunction 실행 후 실행을 멈춘다`가 아니라 `someFunction의 실행 결과를 반환한다`로 인식되기 마련이에요.

읽는 사람으로 하여금 혼란을 야기할 수 있어요!